MAINTAINERS

```diff
@@ -1,4 +1,3 @@
-Michael Crosby <michael@thepasture.io> (@crosbymichael)
```

Comment:
The chief maintainer has to be updated now https://github.com/opencontainers/runc/blob/90ebcca8f105c69eadc98d11733c25b405c23fa2/MAINTAINERS_GUIDE.md?plain=1#L76-L79

Reply:
Oh well. Maybe we can remove this position, since runc is mostly in the maintenance phase, with no major architectural decisions upcoming? WDYT @opencontainers/runc-maintainers ?

Reply:
👍  on removing the Chief Maintainer position, but I'm not sure whether "runc is mostly in the maintenance phase" is true.

Reply:
Yeah, the Chief Maintainer role is something we probably copied from the original maintainers file in Docker / Moby, it's probably fine to remove

Reply:
Added a commit removing the chief maintainer's position.